### DISCUSSION TITLE
Is trusted_domains really "updated at installation time only" with `nextcloud.host`?

### DISCUSSION THREAD

**Discussion by lindhe** [nextcloud/helm#651] [Q&A]
It says [in the README](https://github.com/nextcloud/helm/blob/8d5acefd3aee2cb7c2820f5db92a3ceff1a613ca/charts/nextcloud/README.md?plain=1#L114) that the `nextcloud.host` value "updates trusted_domains at installation time only".

I do not understand why this is at installation time only. It seems like it's set as an environment variable, maybe it's only handled by the application during first initialization and then never again?

The value [`nextcloud.trustedDomains`](https://github.com/nextcloud/helm/blob/8d5acefd3aee2cb7c2820f5db92a3ceff1a613ca/charts/nextcloud/README.md?plain=1#L127) does not have a similar warning.

Basically I'm wondering if it's possible to change `nextcloud.host` after first install or not. Anybody knows?

**Comment by lindhe** [comment]
As far as I can tell, it is true tough. I tried this:

```console
$ kubectl exec -it deploy/new-nextcloud -c nextcloud -- ./occ config:system:get trusted_domains
localhost
nextcloud.kube.home

$ helm upgrade new nextcloud/nextcloud --version 6.1.1 --set 'nextcloud.host=foo.example.com' -f values.yaml

$ kubectl exec -it deploy/new-nextcloud -c nextcloud -- ./occ config:system:get trusted_domains
localhost
nextcloud.kube.home
```

So it seems the description for the value is true; changing that value does not work after first install.
